development/webpack/webpack.config.ts

```diff
@@ -172,9 +162,30 @@ const plugins: WebpackPluginInstance[] = [
       { from: join(context, 'images'), to: 'images' },
       // Copy rive.wasm for Rive animations
       {
-        from: join(context, '../node_modules/@rive-app/canvas/rive.wasm'),
+        from: join(nodeModules, '@rive-app/canvas/rive.wasm'),
         to: 'images/riv_animations/rive.wasm',
       },
+      // snaps MV3 needs the offscreen document
+      ...(MANIFEST_VERSION === 3
+        ? [
+            {
+              from: join(
```

Comment:
Probably better to use `require.resolve(<path>)` instead of `nodeModules` here. I _think_ it should work, even though these files are "exported" via the package's own `package.json`? Could be wrong 🤷  .